ruleset/rules/0755-office365_rules.xml

```diff
@@ -0,0 +1,928 @@
+    </rule>
+
+    <rule id="91545" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^15$</field>
+        <description>Office 365: Secure Token Service (STS) logon events in Azure Active Directory.</description>
+        <options>no_full_log</options>
+        <group>AzureActiveDirectoryStsLogon</group>
+    </rule>
+
+    <rule id="91546" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^16$</field>
+        <description>Office 365: Public Switched Telephone Network (PSTN) events from Skype for Business.</description>
+        <options>no_full_log</options>
+        <group>SkypeForBusinessPSTNUsage</group>
+    </rule>
+
+    <rule id="91547" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^17$</field>
+        <description>Office 365: Blocked user events from Skype for Business.</description>
+        <options>no_full_log</options>
+        <group>SkypeForBusinessUsersBlocked</group>
+    </rule>
+
+    <rule id="91548" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^18$</field>
+        <description>Office 365: Admin actions from the Security and Compliance Center.</description>
+        <options>no_full_log</options>
+        <group>SecurityComplianceCenterEOPCmdlet</group>
+    </rule>
+
+    <rule id="91549" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^19$</field>
+        <description>Office 365: Aggregated Exchange mailbox auditing events.</description>
+        <options>no_full_log</options>
+        <group>ExchangeAggregatedOperation</group>
+    </rule>
+
+    <rule id="91550" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^20$</field>
+        <description>Office 365: Power BI events.</description>
+        <options>no_full_log</options>
+        <group>PowerBIAudit</group>
+    </rule>
+
+    <rule id="91551" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^21$</field>
+        <description>Office 365: Dynamics 365 events.</description>
+        <options>no_full_log</options>
+        <group>CRM</group>
+    </rule>
+
+    <rule id="91552" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^22$</field>
+        <description>Office 365: Yammer events.</description>
+        <options>no_full_log</options>
+        <group>Yammer</group>
+    </rule>
+
+    <rule id="91553" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^23$</field>
+        <description>Office 365: Skype for Business events.</description>
+        <options>no_full_log</options>
+        <group>SkypeForBusinessCmdlets</group>
+    </rule>
+
+    <rule id="91554" level="3">
```

Comment:
Security and Compliance Center events, could it be level 5?

Reply:
Done